sdk/core/core-http/test/expiringAccessTokenCacheTests.ts

```diff
@@ -1,7 +1,10 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
 import { assert } from "chai";
 import { ExpiringAccessTokenCache } from "../src/credentials/accessTokenCache";
 
-function mockToken(expirationDeltaMs: number) {
+function mockToken(expirationDeltaMs: number): any {
```

Comment:
why return `any`? it was returning something much more specific before when it inferred the result shape

Reply:
I can't leave this without return type 🤔

Reply:
Oh! I just realized that what makes sense here is "AccessToken". I'll use that instead.